### ISSUE TITLE
Issues with .getserverlog, advancedproccall, play local sound, etc.

### ISSUE THREAD

**Issue by bryanlavergne** [Apollo-Community/ApolloStation#405]
Having issues with the above verbs. I am not able to get any logs nor play sounds.

**Comment by lmas** [comment]
Works for me, make sure you have the correct rank to use them.

Closing this for now, but feel free to comment again if it's still not working.

**Comment by lmas** [comment]
Could you list all verbs that doesn't work for you?

Edit: So apparently only getserverlog is bugged now.

**Comment by lmas** [comment]
Might be a bug with this line (~~if it's not an issue on the client side~~ nope, guess not since multiple users have this problem):
https://github.com/Apollo-Community/ApolloStation/blob/master/code/modules/admin/verbs/getlogs.dm#L78

**Comment by bryanlavergne** [comment]
King_Nexus just ran into the bug as well with using .getserverlog. He goes through the same symptoms: the log shows that he accesses the file, and he is given a message to wait a few minutes in the case that the file is big. But, there are no signs that a file is sent.

KN managed to get around it by logging out and back in. I tried the same, but did not work.

An additional note, I have been able to access logs recently. But most of the time, I run into this issue.
